### PR TITLE
PR for Sign In Page (Fix: fixed grid breaking at smaller width)

### PR DESCRIPTION
**Pull request by kiranpratyush**
Fixed breaking grid Overflow and added some simple responsiveness to the Sign In page. Please review it

[Sign in page](https://testing2322.netlify.app/pages/signin/)

image
image

- Fix: fixed breaking grid properties for smaller width
- Feat: Added small responsiveness for the Sign In page

### CODE REVIEW

File: pages/SignIn/style.css [hunk 1]
```diff
@@ -12,26 +12,36 @@ label{
...
+    width:100%;
+}
+
+@media screen and (max-width:550px)
+{
+    .auth-main{
+        padding:15px;
+    }
+    .sign-up,.sign-in{
+        width:100%;
```
Comment (msdeshmukh009) [inline pages/SignIn/style.css]: You have already given `.sign-up .sign-in` width of 100% so no need to do it again. Am I missing something?

**Review by msdeshmukh009 — APPROVED** [review]
Rest looks good to me.

**Comment by preethasri** [comment]
hey kiran UI looks good . u can also add the live link for better understanding

**Review by preethasri — APPROVED** [review]

File: pages/SignIn/style.css (hunk 1 above)
Comment (kiranpratyush) [reply pages/SignIn/style.css]: Thank you for pointing out fixed 👍

**Pull request merged**